deploy-source/solutions/expense-client/src/main/java/com/redhat/training/model/Expense.java

```diff
@@ -5,6 +5,8 @@
 import java.time.LocalDateTime;
 import java.util.UUID;
 
+import org.hibernate.annotations.Type;
```

Comment:
Are we missing actually using the `Type` annotation?  We import it but it is not used in the code.